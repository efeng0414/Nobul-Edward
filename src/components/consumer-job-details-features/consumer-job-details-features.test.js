import React from "react";
import renderer from "react-test-renderer";
import ConsumerJobDetailsFeatures from "./index";

describe("ConsumerJobDetailsFeatures", () => {
  const componentProps = {
    intl: global.getIntl(),
    price: {},
    propertyType: "buy",
    regions: {},
    bedrooms: 12,
    bathrooms: 0,
    propertyFeatures: {},
    isLoading: true,
    title: "ConsumerJobDetails",
    jobType: "buy",
    address: "12100 metric blvd",
    description: "my house"
  };
  const componentJSX = <ConsumerJobDetailsFeatures {...componentProps} />;
  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
