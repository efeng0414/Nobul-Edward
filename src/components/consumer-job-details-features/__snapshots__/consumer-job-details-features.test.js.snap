// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConsumerJobDetailsFeatures Component Snapshot should match existing snapshot 1`] = `
<div>
  ConsumerJobDetails
  <div
    className="ant-card interested-in"
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-spin-nested-loading spinner-wrapper"
        style={null}
      >
        <div>
          <div
            className="ant-spin ant-spin-lg ant-spin-spinning ant-spin-show-text"
          >
            <span
              className="ant-spin-dot ant-spin-dot-spin"
            >
              <i />
              <i />
              <i />
              <i />
            </span>
            <div
              className="ant-spin-text"
            >
              Loading......
            </div>
          </div>
        </div>
        <div
          className="ant-spin-container ant-spin-blur"
        >
          <div>
            <div
              className="job-type-price-location"
            >
              <div
                className="job-type-price-location-left"
              >
                <div
                  className="job-type-price-location-item"
                >
                  <h5>
                    Property Type
                  </h5>
                  <span>
                    Buy
                  </span>
                </div>
                <div
                  className="job-type-price-location-item"
                >
                  <h5>
                    Price Range
                  </h5>
                  <span>
                    <span
                      className="currency"
                    >
                      <span
                        className="currency__identifier"
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": undefined,
                          }
                        }
                      />
                    </span>
                  </span>
                </div>
              </div>
              <div
                className="job-type-price-location-right"
              >
                <div
                  className="job-type-price-location-item"
                >
                  <h5>
                    Location
                  </h5>
                  <div
                    className="limited-list"
                  >
                    <ul
                      className="limited-list__list"
                    />
                  </div>
                </div>
                <div
                  className="job-type-price-location-item"
                >
                  <h5>
                    Description
                  </h5>
                  <span>
                    my house
                  </span>
                </div>
              </div>
            </div>
          </div>
          <div
            className="interested-in-icons"
          >
            <div
              className="job-features-icons"
            >
              <h5>
                Features
              </h5>
              <div
                className="ant-row"
              >
                <div
                  className="ant-col-sm-12 ant-col-md-6"
                >
                  <img
                    src="file-mock"
                  />
                  <span>
                    12 Bedrooms
                  </span>
                </div>
                <div
                  className="ant-col-sm-12 ant-col-md-6"
                >
                  <img
                    src="file-mock"
                  />
                  <span>
                    0 Bathrooms
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
